src/Layers/xrRenderPC_R1/FStaticRender.h

```diff
@@ -33,6 +33,7 @@ class CRender : public D3DXRenderBase
         u32 disasm : 1; // config
         u32 forceskinw : 1; // config
         u32 no_detail_textures : 1; // config
+        u32 managed_tex_disabled : 1; // don't keep textures in RAM
```

Comment:
Kinda bad naming. This flag is just a runtime analog of `-no_staging` option.

Reply:
Agree, better name is `no_texture_staging`.

Reply:
Ok